db/migrate/20210728013735_create_users.rb

```diff
@@ -1,9 +1,9 @@
 class CreateUsers < ActiveRecord::Migration[5.2]
   def change
     create_table :users do |t|
-      t.string :uid
       t.string :name
       t.string :email
+      t.string :password_digest
```

Comment:
Password Digest to be removed later

Reply:
This should be resolved now! Merged in from main and this doesn't seem to be here anymore

Reply:
this was fixed by a hotfix yesterday